firefox_ui_tests/remote/security/test_green_larry.py

```diff
@@ -0,0 +1,120 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver.errors import NoSuchElementException
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestGreenLarry(FirefoxTestCase):
```

Comment:
Please rename the test to something more understandable. I would suggest `test_ev_certificate.py`